orangecontrib/network/widgets/OWNxExplorer.py

```diff
@@ -32,7 +32,7 @@ class Inputs:
 
     class Outputs(OWDataProjectionWidget.Outputs):
         subgraph = Output("Selected sub-network", Network)
-        unselected_subgraph = Output("Remaining sub-network", Network)
+        #unselected_subgraph = Output("Remaining sub-network", Network)
```

Comment:
Is there a reason why this is commended out?

Reply:
Yes. Together with the inherited outputs, this widget would have five, which doesn't work until merging https://github.com/biolab/orange-canvas-core/pull/168 and releasing new canvas. :)

Reply:
Actually, I'm gonna merge https://github.com/biolab/orange-canvas-core/pull/168 now, and then uncomment this.

Reply:
I removed the commit that commented this out.